mods/tuxemon/maps/map1.tmx

```diff
@@ -277,20 +277,18 @@
     <property name="cond2" value="is player_at"/>
    </properties>
   </object>
-  <object id="78" name="Player's House Sign" type="event" x="624" y="736" width="16" height="16">
+  <object id="78" name="Player's House Sign" type="event" x="624" y="720" width="16" height="16">
```

Comment:
Why is the Y value changing here (and in other events)?

Reply:
Relevant to the next time - it's because we're changing from 'player is here' position to 'object is here' position, which is 1 up